docs/sources/operators-guide/configuring/configuring-out-of-order-samples-ingestion.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Configuring out-of-order samples ingestion"
+menuTitle: "Configuring out-of-order samples ingestion"
+description: "Learn how to configure Grafana Mimir to handle out-of-order samples ingestion."
+weight: 120
+---
+
+# Configuring out-of-order samples ingestion
+
+Grafana Mimir and the Prometheus TSDB understand out-of-order as follows.
+
+The moment that a new series sample arrives, Mimir need to determine if the series already exists, and whether or not the sample is too old:
+
+- If the series exists, incoming sample must have newer timestamp than the latest sample that is stored for the series.
+  Otherwise, it is considered out-of-order and will be dropped by the ingesters.
+- If the series does not exist, then the sample has to be within bounds, which go back 1 hour from TSDB's head-block max time (when using 2 hour block range). If it fails to be within bounds, then it is also considered out-of-order and will be dropped by the ingesters.
+
+If you have out-of-order samples due to the nature of your architecture or the system that is being observed, then you can configure Grafana Mimir to set an out-of-order time-window threshold for how old samples can be ingested.
```

Comment:
Before this, I would say that If you're remote writing your metrics from Prometheus or Grafana Agent, then out-of-order samples are unexpected, because Prometheus and Grafana Agent guarantee that samples are written in-order for each series.

Reply:
Thats true for a single instance of each right? In the event that you had multiple prometheus remote writting the same series would that still hold true?

Reply:
> In the event that you had multiple prometheus remote writting the same series would that still hold true?

In that situation you would see out of order samples, but then you should probably fix your Prometheus configuration in such a way that each prometheus produces unique series

Reply:
Added a note block clarifying this.

Thanks @replay I assumed that would be the case but then I thought, maybe there is a usecase where the customer don't want to add labels to differentiate the source that originates and only wants the datapoints :shrug: